src/inmanta/server/protocol.py

```diff
@@ -504,6 +526,14 @@ def __init__(self) -> None:
         # Listeners
         self.listeners: List[SessionListener] = []
 
+    async def get_status(self) -> Dict[str, ArgumentTypes]:
+        return {
+            "hangtime": self.hangtime,
+            "interval": self.interval,
+            "sessions": len(self._sessions),
+            "listeners": len(self.listeners),
```

Comment:
this is a rather deep down internal detail, don't know if this makes sense.
Also the number is not that interesting to know, without known what is listening